subsys/bluetooth/audio/ascs.c

```diff
@@ -2653,3 +2647,10 @@ static int control_point_notify(struct bt_conn *conn, const void *data, uint16_t
 }
 
 #endif /* BT_AUDIO_UNICAST_SERVER */
+
+#if defined(ZTEST_UNITTEST)
+const struct bt_gatt_service_static *ascs_get_gatt_service(void)
+{
+	return &ascs_svc;
+}
+#endif /* ZTEST_UNITTEST */
```

Comment:
Hmm, generally we avoid having test-specific code in the stack, but maybe it's inevitable

Reply:
Hmm, perhaps `bt_gatt_foreach_attr` can be used for this instead?

Reply:
As this is compiled out of kernel, I decided provide test doubles for GATT. The GATT itself has many kernel dependencies I did not want to mock. I.e here is no `bt_gatt_foreach_attr` function compiled, so I needed to provide a mock for it.